workflow/sample_files/base-hvac-mini-split-heat-pump-ducted-detailed-performance-autosize.xml

```diff
@@ -437,6 +443,24 @@
                     <Value>1.93</Value>
                   </Efficiency>
                 </PerformanceDataPoint>
+                <PerformanceDataPoint>
+                  <OutdoorTemperature>-5.0</OutdoorTemperature>
```

Comment:
This NEEP product has data at -5F too, so I added it. (I also removed the `base-hvac-air-to-air-heat-pump-var-speed-detailed-performance-other-temperatures.xml` file, since we're now testing an additional temperature here).